examples/systemcalls/systemcalls.c

```diff
@@ -16,6 +22,19 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success
  *   or false() if it returned a failure
 */
+	int status;
+	status = system(cmd);
+//Check if the system call is successful and return false on failure
+	if ((status == -1))
+	{
+		return false;
```

Comment:
@charan1811 what would happen if the `system` call returned an error code other than -1?